mlflow/langchain/api_request_parallel_processor.py

```diff
@@ -336,6 +365,64 @@ def _try_transform_response_to_chat_format(response):
         else:
             return transformed_response.model_dump(mode="json")
 
+    @staticmethod
+    def _try_transform_response_iter_to_chat_format(chunk_iter):
```

Comment:
Should this be serialized JSON as a return type or a ChatChunkResponse obj?

Reply:
I think we should make it consistent with `_try_transform_response_to_chat_format` return value